pkg/image/config.go

```diff
@@ -0,0 +1,25 @@
+package image
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/replicate/cog/pkg/config"
+	"github.com/replicate/cog/pkg/docker"
+)
+
+func GetConfig(imageName string) (*config.Config, error) {
+	image, err := docker.ImageInspect(imageName)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to inspect %s: %w", imageName, err)
+	}
+	configString := image.Config.Labels["org.cogmodel.config"]
+	if configString == "" {
+		return nil, fmt.Errorf("Image %s does not appear to be a Cog model", imageName)
```

Comment:
I wonder if it should also check Cog version here for future compatibility?

"You are running Cog version X but this image was produced by Cog version Y. You need to upgrade Cog to run this image: ..."

It might be annoying for it to happen for every patch version while we're in `0.0.X`, so maybe it triggers at `0.1.0` and we can figure out a better versioning scheme then.

Reply:
I agree, but perhaps that should be a different PR. I'll open an issue for now so we don't forget.